go.mod

```diff
@@ -1,25 +1,27 @@
 module github.com/fluxcd/image-automation-controller
 
-go 1.14
+go 1.15
```

Comment:
Why is go 1.15 required?

Reply:
A: it is not specifically required; however, since it's difficult to avoid providing _some_ value here, and in the absence of any version-specific features being needed, it's better to make it match the version used in e.g., CI.